torch/_inductor/scheduler.py

```diff
@@ -180,7 +182,7 @@ def used_or_aliased_buffer_names(self) -> Set[str]:
                 # needed to avoid deallocating aliased buffer
                 # if there are still uses of aliases ahead
                 if isinstance(layout, ir.AliasedLayout):
-                    used_names.add(layout.view.data.get_name())
+                    used_names.add(layout.view.get_name())
```

Comment:
hm why this change

Reply:
Otherwise, there will be an error "IRNode has no attribute data".

Reply:
Updated view type hints in ir.py instead..